nolabs/api_models/biobuddy.py

```diff
@@ -62,8 +65,10 @@ class FunctionCall:
     parameters: List[FunctionParam]
     data: FunctionCallReturnData = None
 
+
 @pcdataclass.dataclass
 class Message:
+    id: str
     role: str
```

Comment:
Create a string Enum and use it here instead of arbitrary strings. You will be able to check values of role in functions and also fastapi swagger will show it in docs automatically